kod/object/active/holder/nomoveon/battler/monster/towns/marntwn/MrSmith.kod

```diff
@@ -92,7 +92,8 @@ messages:
 
    UserEntered(who = $)
    {
-      Post(poOwner,@SomeoneSaid,#what=self,#string=MrBs_entry_welcome,
+
+      Send(poOwner,@SomeoneSaid,#what=self,#string=MrBs_entry_welcome,
```

Comment:
Just verifying that you have tested every one of these instances where you changed Post to Send.  It's possible some were added intentionally to get the order of messages correct upon entering a room.  Please revert any you haven't tested.